Source/Android/app/src/main/res/values/arrays.xml

```diff
@@ -306,6 +306,19 @@
         <item>Right Stick</item>
     </string-array>
 
+    <string-array name="doubleTap">
+        <item>Wiimote A</item>
+        <item>Wiimote B</item>
+        <item>Wiimote 2</item>
+    </string-array>
+
+    <string-array name="doubleTapWithClassic">
+        <item>Wiimote A</item>
+        <item>Wiimote B</item>
+        <item>Wiimote 2</item>
```

Comment:
We use "Wii Remote" in all user-facing strings, not "Wiimote".

Reply:
Fixed